include/assimp/mesh.h

```diff
@@ -129,27 +129,24 @@ struct aiFace
     //! The maximum value for this member is #AI_MAX_FACE_INDICES.
     unsigned int mNumIndices;
 
-    //! Pointer to the indices array. Size of the array is given in numIndices.
-    unsigned int* mIndices;
```

Comment:
If I change it to be a pointer that references the indices array in the mesh, will this PR be accepted? This solution works for us, and I'm happy to make the change if that's the only thing stopping it from being accepted, but I've already spent way too much time working on this.